ingeniamotion/wizard_tests/feedbacks_tests/feedback_test.py

```diff
@@ -205,6 +205,10 @@ def feedback_setting(self) -> None:
         self.feedback_resolution = self.mc.configuration.get_feedback_resolution(
             self.sensor, servo=self.servo, axis=self.axis
         )
+        if self.feedback_resolution == 0:
+            raise ValueError(
```

Comment:
@MartinALL91 Because of another issue today we have talked about error handling on motionlab.
If this is a generic exception it will be reported on rollbar. Shouldn't it be a specific IM exception type that  is not catched by rollbar?

Reply:
Done